pkg/debug/checkups/root_directory.go

```diff
@@ -0,0 +1,58 @@
+package checkups
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type RootDirectory struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+}
+
+func (c *RootDirectory) Name() string {
+	return "Root directory contents"
+}
+
+func (c *RootDirectory) Run(_ context.Context, fullFH io.Writer) error {
+	files, err := os.ReadDir(c.k.RootDirectory())
+	switch {
+	case errors.Is(err, os.ErrNotExist):
+		c.status = Failing
+		c.summary = fmt.Sprintf("root directory (%s) not present", c.k.RootDirectory())
+	case err != nil:
+		c.status = Erroring
+		c.summary = fmt.Sprintf("listing files in root directory (%s): %s", c.k.RootDirectory(), err)
+	case len(files) == 0:
+		c.status = Warning
+		c.summary = fmt.Sprintf("root directory (%s) empty", c.k.RootDirectory())
+	default:
+		c.status = Passing
+		c.summary = fmt.Sprintf("root directory (%s) contains %d files", c.k.RootDirectory(), len(files))
```

Comment:
Would we potentially want to check for the existence of some of the more important files within this directory too?

Reply:
I was thinking launcher.db/osquery.db, and maybe it'd be interesting to know if the `updates/` directory exists and what's in it. `updates/` at least sounds like it would belong in another checkup.

Reply:
updates feels like another checkup.

I was thinking about a checkup specific to the DBs. Do they exist, how big, if we have access, what can we say about it?

Reply:
All of that sounds helpful to me, feel free to punt any of that forward to another PR